app/models/searchgov_url.rb

```diff
@@ -34,14 +34,14 @@ class SearchgovUrl < ActiveRecord::Base
     column_name: proc {|url| !url.fetched? ? 'unfetched_urls_count' : nil },
     column_names: { ['searchgov_urls.last_crawled_at IS NULL'] => 'unfetched_urls_count' }
 
-  scope :fetch_required, -> { where('last_crawled_at IS NULL OR lastmod > last_crawled_at') }
+  scope :fetch_required, -> { where('last_crawled_at IS NULL OR lastmod > last_crawled_at OR enqueued_for_reindex') }
```

Comment:
Let's wrap this long line. See [the style guide](https://github.com/rubocop-hq/ruby-style-guide#lambda-multi-line) for multi-line lambda styling.

Reply:
Please see the [style guide](https://github.com/rubocop-hq/ruby-style-guide#lambda-multi-line) for multi-line lambda styling.

Reply:
@peggles2 , the style guide recommends:
```ruby
l = lambda do |a, b|
  tmp = a * 7
  tmp * b / 50
end
```

Reply:
@peggles2 ^^